server/handler/check_run.go

```diff
@@ -56,8 +59,16 @@ func (h *CheckRun) Handle(ctx context.Context, eventType, deliveryID string, pay
 
 	prs := event.GetCheckRun().PullRequests
 	if len(prs) == 0 {
-		logger.Debug().Msg("Doing nothing since status change event affects no open pull requests")
-		return nil
+		logger.Debug().Msg("No pull requests associated with the check run, searching by SHA")
+		// if no PR's were attached with the event let's check with Github in case it is a fork
```

Comment:
It might be good to clarify here that check runs for fork PRs will always have an empty pull request list. I think the other case is check runs on branches that aren't part of any PR?